parcel_test.go

```diff
@@ -31,57 +32,104 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+
+	require.NoError(t, err)
+
+	defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	id, err := store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
 
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
+	p, err := store.Get(id)
+	require.NoError(t, err)
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	pNum0 := Parcel{0, p.Client, p.Status, p.Address, p.CreatedAt}
```

Comment:
Можно чуть проще, твой вариант не масштабируемый, так как при изменении структуры нужно переписывать эту строчку
```suggestion
	p.Number = 0
```

Reply:
Это же так логично, я все пытался перемудрить))